bedrock/products/views.py

```diff
@@ -27,6 +27,21 @@ def vpn_landing_page(request):
     sub_not_found = request.GET.get('vpn-sub-not-found', None)
     locale = l10n_utils.get_locale(request)
     pricing_params = settings.VPN_VARIABLE_PRICING.get(locale, settings.VPN_VARIABLE_PRICING['us'])
+    cta_experiment = request.GET.get('cta_experiment', None)
+    cta_variation = request.GET.get('cta_variation', None)
```

Comment:
This should use `entrypoint_variation`
```suggestion
    entrypoint_variation = request.GET.get('entrypoint_variation', None)
```